fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java

```diff
@@ -150,6 +151,12 @@ public Void visitUpdateStatement(UpdateStmt node, Void context) {
             return null;
         }
 
+        @Override
+        public Void visitDeleteStatement(DeleteStmt node, Void context) {
+            getDB(node.getTableName());
```

Comment:
If we support delete/update statements references multiple tables, there shoud be also changed. Can you add a TODO or comment on the UpdateStmt/DeleteStmt.class to hlep people remember change this class at the same time.

Reply:
done